### ISSUE TITLE
Gradually increase default results limit for /map view

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#10123]
In #10059, we made it possible to increase the results limit at https://publiclab.org/map using the parameter `limit`:

https://publiclab.org/map?limit=100

I just adjusted the default to 25. We'll ratchet it upwards gradually and take notes:

* [x] 25
* [ ] 50
* [ ] 100

The code is here:

https://github.com/publiclab/plots2/blob/38ab2988fbd3e63227d02bd5398d64cc957d2279/app/views/map/map.html.erb#L95

### Limit 100

Manually setting the limit to 100 and browsing around a bit did show a bit of a spike in Skylight:

![Screen Shot 2021-08-31 at 12 06 53 PM](https://user-images.githubusercontent.com/24359/131537665-e708ef00-63dd-42e8-995e-65f0124feae3.png)

`nearbyPeople` not bad at max of 1.8s, but `taglocations` worse at up to 14s:

![Screen Shot 2021-08-31 at 12 07 35 PM](https://user-images.githubusercontent.com/24359/131537773-3a59b597-cbfa-450d-b910-4d3534cf4785.png)
![Screen Shot 2021-08-31 at 12 07 39 PM](https://user-images.githubusercontent.com/24359/131537777-581ecc02-6b7b-48e5-8870-ca64570ff8b1.png)

We optimized tagLocations in #9946/https://github.com/publiclab/plots2/pull/10028, thanks to @17sushmita. I wonder if we could go further since it's going quite slow still at up to 14s.

We could look into partitioning calls into whole-integer-latitude/longitude boxes... as mentioned also in https://github.com/publiclab/plots2/issues/9946#issuecomment-898941171

**Comment by 17sushmita** [comment]
Yes, I think we can try some more optimizations as I mentioned here https://github.com/publiclab/plots2/pull/10028#issuecomment-899899653. Also, looking into https://github.com/publiclab/plots2/issues/9946#issuecomment-898941171

**Comment by jywarren** [comment]
Went to limit of 50 in https://github.com/publiclab/plots2/commit/032edf8708e779e712a3fe52416eb3cc6346f3f9 !! 🤞

**Comment by jywarren** [comment]
Publishing to live now.

**Comment by jywarren** [comment]
OK, so at default limit 50, we are seeing a lot of great content (and the MapKnitter CORS fix shows maps now too):

![image](https://user-images.githubusercontent.com/24359/134075905-a01beec6-3f4c-4af3-98c2-aaa5e26a7995.png)

vs limit 100: 

![image](https://user-images.githubusercontent.com/24359/134075940-73c74cb0-54da-4c9e-8da8-48549b1fcfb9.png)


vs the prior limit of 5 (i think):

![image](https://user-images.githubusercontent.com/24359/134075852-4a1e3a7f-2dd9-481f-b024-f53c971e8af2.png)


yet it's not hitting the server too hard! I'm checking Skylight now but I think this means we are OK for uptime with the default limit 50, and don't need to pull back. 

![image](https://user-images.githubusercontent.com/24359/134075527-f9c0aa16-1c6b-415f-9c23-5addd51609cf.png)

**Comment by jywarren** [comment]
It appears our tagLocations endpoint is doing even better! 

![image](https://user-images.githubusercontent.com/24359/134076681-8750dd0c-2218-4877-be93-f6f98d6129fd.png)

**Comment by jywarren** [comment]
whereas nearbyPeople does occasionally get a 23 second load time: 

![image](https://user-images.githubusercontent.com/24359/134076743-07c3fe29-89c6-4c63-89f5-af7c973d9065.png)

**Comment by jywarren** [comment]
I'm comfortable leaving things as they are for a while pending more exploration, experimentation, and input. I think there's a lot of mileage to be had from turning other default layers on, perhaps, or doing more to tune/trim and interlink MapKnitter markers nearby, and/or tune the display of people, maybe? I think limits are doing OK right now at 50.
